lib/Text/Glob.pm

```diff
@@ -177,6 +189,10 @@ the regex, or change the seperator that Text::Glob uses by setting
 C<$Text::Glob::seperator> to an alternative value while compiling the
 the regex.
 
+However, if C<$Text::Glob::globstar> is set to a true value, two
+consecutive asterisks (e.g. C<**.foo>) B<will> match C</>, though one
+asterisk will not.
+
```

Comment:
The intent (and, I **believe** the functionality) of the patch was to allow something like '**.foo' to match 'baz/burfle.foo'. Without it (or without turning it on) the second asterisk is redundant, and though 'burfle.foo' would still match, 'baz/burfle.foo' would not. Looking at it now I can imagine a further tweak to collapse consecutive slashes, so that, for example, '/usr/**/foo' would match any and all of '/usr/local/bin/foo', '/usr/bin/foo', or '/usr/foo'. I believe that with the patch as it now stands '/usr/**/foo' would match the first two but not the third.